extra/cloudformation/default-unauth-resources-template.yaml

```diff
@@ -698,13 +698,17 @@ Resources:
                                   responseData = {Error: "describeEndpoint call failed"};
                                   console.log(responseData.Error + ":\n", err);
                             } else {
+                                  const splitEndpointAddress = data.endpointAddress.split(".");
+                                  const endpointAddress = !splitEndpointAddress[0].endsWith("-ats")
+                                        ? `${splitEndpointAddress[0]}-ats.${splitEndpointAddress
+                                                .splice(1)
+                                                .join(".")}`
+                                        : data.endpointAddress;
```

Comment:
Actually, looking at this further, I don't understand the logic here. From the IoT docs, it seems like the "-ats" is distinguishing between Verisign-signed and ATS-signed endpoints: https://docs.aws.amazon.com/iot/latest/apireference/API_DescribeEndpoint.html . If the goal is to always have ats-signed endpoints, shouldn't this be passing in endpointType = iot:Data-ATS? I'm not sure it's guaranteed that you can take a Verisign-signed endpoint and just tack "-ats" at the end to get an ATS-signed endpoint.

Reply:
@mbalfour-amzn closing this, raised another [PR](https://github.com/aws-geospatial/amazon-location-features-demo-web/pull/247) that is updated with main, and this resolved your above comment.